shade_janitor/cleanup.py

```diff
@@ -1,65 +1,136 @@
 #!/usr/bin/env python
 
+def cleanup_instances(instances):
```

Comment:
Don't you need to pass cloud into this function for it to be able to do it's work?